src/astra_driver.cpp

```diff
@@ -44,6 +44,8 @@
 #include <boost/date_time/posix_time/posix_time.hpp>
 #include <boost/thread/thread.hpp>
 
+using diagnostic_msgs::DiagnosticStatus;
```

Comment:
You are not taking advantage of this. Either remove the `using` and the `diagnostic_msgs` instances in the code.

Reply:
still to be addressed?

Reply:
No, it is changed. Just not in this line of code. I left `using` and removed `diganostic_msgs::`